_rules/SC2-1-2-no-keyboard-trap.md

```diff
@@ -21,9 +22,9 @@ authors:
 
 ### Applicability
 
-The rule applies to any HTML or SVG element on a web page that is [focusable](#focusable).
+The rule applies to any HTML or SVG element that is [focusable](#focusable).
 
-Note: The WCAG 2.0 success criterion 2.1.2 applies to all content, not just HTML or SVG, where focus can be moved to through keyboard navigation.
+**Note**: This rule is only applies to HTML and SVG. Thus, it is a partial check for WCAG 2.0 success criterion 2.1.2, which applies to all content.
```

Comment:
Given that this is true for _most_ of the rules we write, I'd question the value of this note. For the sake of consistency, it should either be left out or included in _all_ rules where this applies.

Reply:
We are missing somewhere to document the interpretation of the success criterion that says something about what the success criterion covers, what the test rule(s) cover(s), and whether the test rule(s) only cover(s) part of the success criterion.

For the time being, the applicability section of the test rule is the best place suited. We are in favour of having a separate document where this is described. 

For the sake of consistency, we agree that this should be included in all test rules where this applies.

Reply:
Grammar point - "This rule only applies..." or "This rule is only applicable..." 

What other content would WCAG apply to? In a webpage HTML elements like canvas or video or form element are often heavily Javascript related.

Reply:
Agreed @EmmaJP